src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/db/ActiveOperationComponent.scala

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.dsde.workbench.gpalloc.db
+
+import org.broadinstitute.dsde.workbench.gpalloc.model.BillingProjectStatus
+import org.broadinstitute.dsde.workbench.gpalloc.model.BillingProjectStatus.BillingProjectStatus
+
+case class ActiveOperationRecord(billingProjectName: String,
+                                 operationType: String,
+                                 operationId: String,
+                                 done: Boolean,
+                                 errorMessage: Option[String])
+
+trait ActiveOperationComponent extends GPAllocComponent {
```

Comment:
How do we mock the Slick DAO operations for testing? I wonder if declaring the operations (e.g. `findOperations`) in `GpAllocComponent` trait would make that easier.

Reply:
we don't -- we spin up a mysql for unit testing